src/main/java/frc/team2412/robot/commands/intake/AllInRumbleCommand.java

```diff
@@ -0,0 +1,23 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.team2412.robot.Controls;
+import frc.team2412.robot.subsystems.IntakeSubsystem;
+
+public class AllInRumbleCommand extends SequentialCommandGroup {
+	private final IntakeSubsystem intakeSubsystem;
+	private final Controls controls;
+
+	public AllInRumbleCommand(IntakeSubsystem intakeSubsystem, Controls controls) {
+		this.intakeSubsystem = intakeSubsystem;
+		this.controls = controls;
+		addRequirements(intakeSubsystem);
```

Comment:
This implicitly happens in the `addCommands` call below (see https://github.com/wpilibsuite/allwpilib/blob/f1a1ffd7fc96649df2293e6337e02f25b557682c/wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/SequentialCommandGroup.java#L51 ) so should be removed here